src/components/product/ProductDetail.js

```diff
@@ -0,0 +1,295 @@
+                  />
+                </div>
+              );
+            })}
+        </div>
+        <div className="main-image ml-2">
+          <img alt="" className="responsive-image" src={mainImage} />
+        </div>
+      </div>
+      <div className="product-description">
+        <h1 className="detail-product-name">{product.name}</h1>
+        <h2 className="detail-product-desc mt-05">{product.description}</h2>
+        <div className="detail-rating flex align-center mt-05">
+          <span
+            className={`review ${
+              product.rating >= 4
+                ? "bgAlertGreen"
+                : product.rating >= 3
+                ? "bgAlertYellow"
+                : "bgAlertRed"
+            } colorWhite m-0-05 border-round`}
+          >
+            <span className="review-text">{product.rating}</span>
+            <i className="fas fa-star"></i>
+          </span>
+          <p className="m-0-1 font-size-sm">{product.ratings} Ratings</p>
+          {product.tags[0] !== "" && (
+            <span class="detail-tag border-round">{product.tags[0]}</span>
+          )}
+        </div>
+        <div className="detail-price mt-05">
+          <div className="flex align-center mt-02">
+            <p className="">MRP :</p>
+            <span className="ml-05 detail-price-mrp price-cut font-size-m">
+              ₹ {product.mrp}
+            </span>
+          </div>
+          <div className="flex align-center mt-02">
+            <p>Price : </p>
+            <span className="ml-05 detail-price-actual">₹ {product.price}</span>
+          </div>
+          <div className="flex align-center mt-02">
+            <p>You Save :</p>
+            <span className="ml-05 detail-price-discount discount">
+              {`₹ ${product.mrp - product.price} (${Math.floor(
+                ((product.mrp - product.price) * 100) / product.mrp
+              )}%)`}
+            </span>
+          </div>
+          <div className="flex align-center mt-02">
+            <p></p>
+            <span className="ml-05 detail-price-actual">
+              Inclusive of all taxes
+            </span>
+          </div>
+        </div>
+        <p className="colorAlertGreen font-size-m mt-05">
+          {product.inStock && "In Stock"}
+        </p>
+        <p className="colorAlertBlue font-size-sm mt-05">
+          {product.freeDelivery && "Free delivery"}
+        </p>
+        <span className="review label font-size-sm bgLightBlue colorBlack mt-05 border-round">
+          {product.fastDelivery ? "Super Fast Shipping" : "Regular Shipping"}
+        </span>
+        <div className="detail-buttons flex align-center">
+          {isInCart === true ? (
+            <Link to="/cart">
+              <button className="btn btn-col btn-outline btn-primary mt-05 border-round">
+                Go to Cart
+                <i className="fas fa-arrow-right ml-05"></i>
+              </button>
+            </Link>
+          ) : (
```

Comment:
instead of
```
 {isInCart === true ? (
            <Link to="/cart">
              <button className="btn btn-col btn-outline btn-primary mt-05 border-round">
                Go to Cart
                <i className="fas fa-arrow-right ml-05"></i>
              </button>
            </Link>
```

this will also work

```
 {isInCart ? (
            <Link to="/cart">
              <button className="btn btn-col btn-outline btn-primary mt-05 border-round">
                Go to Cart
                <i className="fas fa-arrow-right ml-05"></i>
              </button>
            </Link>
```